stan/math/prim/prob/pareto_type_2_lpdf.hpp

```diff
@@ -67,9 +67,10 @@ return_type_t<T_y, T_loc, T_scale, T_shape> pareto_type_2_lpdf(
     }
   }
 
-  VectorBuilder<!is_constant_all<T_shape>::value, T_partials_return, T_shape>
+  VectorBuilder<include_summand<propto, T_shape>::value, T_partials_return,
+                T_shape>
       inv_alpha(size(alpha));
-  if (!is_constant_all<T_shape>::value) {
+  if (include_summand<propto, T_shape>::value) {
     for (size_t n = 0; n < stan::math::size(alpha); n++) {
       inv_alpha[n] = 1 / value_of(alpha_vec[n]);
```

Comment:
This is unrelated, but I'd probably change this to ```inv``` or ```1.0 / ``` to avoid accidentally invoking an integer division